partner_compassion/models/partner_compassion.py

```diff
@@ -11,6 +11,9 @@
 import tempfile
 import uuid
 
+from odoo.addons.website_event_compassion.models.\
```

Comment:
The problème is that `partner_compassion` doesn't depend on module `website_event_compassion`, it is in fact a parent module! Therefore you cannot call methods of this module from there. You should move the method here and do the opposite in `website_event_compassion`. You can as well make the method public since it's needed outside of the module.

Reply:
Corrected with commit 592de88